servers/testkube/server.yaml

```
@@ -0,0 +1,49 @@
+name: testkube
+image: mcp/testkube
+type: server
+meta:
+  category: devops
+  tags:
+    - devops
+    - testing
+    - automation
+    - orchestration
+    - kubernetes
+    - gitops
+    - playwright
+    - cypress
+    - selenium
+    - k6
+    - postman
+    - jmeter
+
+about:
+  title: Testkube MCP Server
+  description: "The Testkube MCP Server exposes continuous testing capabilities (test orchestration, execution, troubleshooting and anlysis) to AI tools and workflows "
+  icon: https://avatars.githubusercontent.com/u/82541796?v=4
+source:
+  project: https://github.com/kubeshop/testkube
+  branch: main
```

As part of our updated security we do need to specify a commit hash in order for us to "pin" the server commit.  Since this was a much older PR the templates didn't provide this. We would need to add a commit hash like this:
```suggestion
  branch: main
  commit: 6da58c790ecffdec93de52f6bcd671317a7e2f5c
```
Assuming we want to use the latest [commit](https://github.com/kubeshop/testkube/commit/6da58c790ecffdec93de52f6bcd671317a7e2f5c) on main